generate_wrappers.py

```diff
@@ -385,12 +396,13 @@ def generate_code(op, api_def, enum_store):
   return_name_and_types = [
       (swiftified_name(a.name), arg_def_type_as_string(a))
       for a in op.output_arg]
+  has_arrow = ' -> ' if len(return_name_and_types) > 0 else ''
```

Comment:
Could we avoid this `has_arrow` variable by including the arrow string in `return_type` directly ?

Reply:
Done.